pii_recognition/evaluation/metrics_test.py

```diff
@@ -1,8 +1,11 @@
 import numpy as np
-from .metrics import compute_f_beta
+from numpy.testing import assert_almost_equal
+
+from .metrics import compute_f_beta, compute_label_precision, compute_label_recall
 
 
 def test_compute_f_beta():
+    # TODO: break into separate test functions
```

Comment:
why not use `sklearn.metrics.fbeta_score`?

Reply:
Make sense. Will put another PR switching to sklearn function. Back then I was thinking of reducing the dependencies when implementing the evaluation make it not relying on many libraries, but since we are going to use sklearn anyway, it won't make any difference.

Reply:
Had another look, might not be able to switch to fbeta_score. First, in previous research where it gets applied we do not have `y_true` and `y_pred` arrays instead we have `Counter` for calculation. Second, there are customised conditions, depending on the values of precision and recall strategies vary.